include/common/nas_structs.h

```diff
@@ -411,6 +411,86 @@ typedef struct tai_list {
 	struct TAI partial_list[EMM_MAX_TAI_LIST];
 } tai_list;
 
+/* 24.301 - 9.9.4.3.1 */
+typedef struct eps_qos_bitmap {
```

Comment:
structure name suggests bit map..but I see all are 1 byte length fields.

Reply:
Moved the fields into struct eps_qos_t.